hvplot/converter.py

```diff
@@ -819,6 +826,17 @@ def _process_data(self, kind, data, x, y, by, groupby, row, col,
             data, x, y, by_new, groupby_new = process_xarray(
                 data, x, y, by, groupby, use_dask, persist, gridded,
                 label, value_label, other_dims, kind=kind)
+            if robust:
+                # adapted from xarray
+                # https://github.com/pydata/xarray/blob/main/xarray/plot/utils.py#L729
```

Comment:
Nit but I prefer GH permalinks.

```suggestion
                # https://github.com/pydata/xarray/blob/6af547cdd9beac3b18420ccb204f801603e11519/xarray/plot/utils.py#L729
```